src/RapidezStatamicServiceProvider.php

```diff
@@ -189,6 +192,21 @@ public function bootPublishables() : self
         return $this;
     }
 
+    public function bootUserModel() : static
+    {
+        if (!class_exists('\App\Models\User::class')) {
+            config(['auth.providers.users.model' => User::class]);
+        }
+
+        if (!Schema::hasTable('users') && !file_exists(database_path('migrations/0001_01_01_000000_create_users_table.php'))) {
```

Comment:
You don't want to have database actions within the service provider as this runs even when you didn't have this setup yet for example with composer

Reply:
Removed the hasTable check, it's not really needed for this anyways. Checking if the migration file exists is enough.